### ISSUE TITLE
Larger requests break JSON parsing

### ISSUE THREAD

**Issue by tolbrino** [hoprnet/derp#2]
These requests are properly forwarded to the underlying RPC endpoint. However, when parsing the body for logging it breaks within the `ClientLogs` durable object context. The issue might be related to the requests being forwarded to the context in an incomplete manner.

**Comment by mjadach-iv** [comment]
@tolbrino can give an example of the calls in question?

**Comment by tolbrino** [comment]
> @tolbrino can give an example of the calls in question?

No. Most likely there is some internal buffer which doesn't fit the entire JSON object and therefore the receiving end cannot parse it. The bug might be outdated though.
